packages/@okta/vuepress-site/docs/reference/api/oauth-clients/index.md

```diff
@@ -769,7 +769,7 @@ Client Applications have the following properties:
 | client_name                             | Human-readable string name of the client application                                                                         | String                                                                                         | FALSE      | FALSE     | FALSE     |
 | client_secret                           | OAuth 2.0 client secret string (used for confidential clients)                                                               | String                                                                                         | TRUE       | TRUE     | TRUE      |
 | client_secret_expires_at                | Time at which the client_secret will expire or 0 if it will not expire(measured in unix seconds)                             | Number                                                                                         | TRUE       | FALSE    | TRUE      |
-| grant_types                             | Array of OAuth 2.0 grant type strings. Default value: `authorization_code`                    | Array of `authorization_code`, `implicit`, `password`, `refresh_token`, `client_credentials`, `urn:ietf:params:oauth:grant-type:saml2-bearer`<ApiLifecycle access="ea" />   | TRUE       | FALSE    | FALSE     |
+| grant_types                             | Array of OAuth 2.0 grant type strings. Default value: `authorization_code`                    | Array of `authorization_code`, `client_credentials`, `implicit` `interaction_code` <ApiLifecycle access="ie" />, `password`, `refresh_token`, `urn:ietf:params:oauth:grant-type:device_code`, `urn:ietf:params:oauth:grant-type:jwt-bearer`, `urn:ietf:params:oauth:grant-type:saml2-bearer` <ApiLifecycle access="ea" />, `urn:ietf:params:oauth:grant-type:token-exchange`   | TRUE       | FALSE    | FALSE     |
```

Comment:
`jwt-bearer` is not supported today, and I'm unsure about the doc status of `interaction_code`. @larsjohansen-okta are we looking to be doc'ing the interaction code grant?

Reply:
It's an OIE feature only, however we do document it here https://developer.okta.com/docs/concepts/interaction-code/ so I believe we should have it as part of the list, perhaps we should make a note it's for OIE only?

Reply:
@larsjohansen-okta - right next to it I have included the Identity Engine flag. That's what we use to indicate that something is OIE only in the docs.

Reply:
Yeah, since we have the other doc, we should be fine with this as well